build-ps/percona-xtradb-cluster.spec

```diff
@@ -906,11 +907,16 @@ if [ -x %{_sysconfdir}/init.d/mysql ] ; then
 fi
 
 %post -n Percona-XtraDB-Cluster-server%{product_suffix}
-
 if [ X${PERCONA_DEBUG} == X1 ]; then
         set -x
 fi
 
+ver=`/bin/rpm -qi percona-xtrabackup | grep Version | awk '{print $3}'`
+if [ $ver = '2.3.5' ]; then
```

Comment:
Maybe it would be better to check only for "2.3" and not the whole number, users don't have to upgrade right away so the version in repository can change and although not likely but possible we will push "2.3.6" and it will not work in this case.